docs/assets/orange-style.css

```diff
@@ -239,12 +260,27 @@
   line-height: var(--font-line-height-display-small);
   letter-spacing: var(--font-letter-spacing-display-small);
 }
-.main-content :is(h2, h3) {
+h2, h3 {
   font-size: var(--font-size-heading-large);
   line-height: var(--font-line-height-heading-large);
   letter-spacing: var(--font-letter-spacing-heading-large);
 }
 
+/* deprecation content */
+.deprecation-content {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    margin-left: 0;
+    margin-right: 0;
+    border: none;
+    padding: 1rem;
+    background-color: var(--bs-color-surface-status-negative-muted);
```

Comment:
As deprecation sounds more like a warning than something prohibited to me, what do you think about changing this color to `surface/status/warning/muted`?

<img width="1084" height="798" alt="Capture d’écran 2025-10-01 à 15 42 44" src="https://github.com/user-attachments/assets/a2ad2faf-c777-4abb-b8dc-235547a77f69" />